main.go

```diff
@@ -2,56 +2,52 @@ package main
 
 import (
 	"fmt"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldlog"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
-	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
-	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
-	"gopkg.in/launchdarkly/go-server-sdk.v5/ldcomponents"
 	"os"
 	"time"
+
+	"gopkg.in/launchdarkly/go-sdk-common.v2/lduser"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
```

Comment:
We have fewer package imports now that we're not doing any custom configuration (which required `ldcomponents` and `ldlog`), and not setting and custom user attributes (which required `ldvalue`).